src/backend/api/Fusion.Resources.Domain/Models/QueryDepartment.cs

```diff
@@ -1,25 +1,58 @@
 using Fusion.Integration.Profile;
 using Fusion.Services.LineOrg.ApiModels;
+using System;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace Fusion.Resources.Domain
 {
     public class QueryDepartment
     {
-        public QueryDepartment(ApiDepartment lineOrgDepartment, FusionPersonProfile? manager)
+        public QueryDepartment(ApiOrgUnitBase lineOrgUnit)
         {
-            DepartmentId = lineOrgDepartment.FullName;
-            LineOrgResponsible = manager;
-        }
+            FullDepartment = lineOrgUnit.FullDepartment;
+            Identifier = lineOrgUnit.SapId;
+            Name = lineOrgUnit.Name;
+            ShortName = lineOrgUnit.ShortName;
+
+            if (lineOrgUnit.Management is not null)
+            {
+                var profile = lineOrgUnit.Management.Persons.FirstOrDefault();
```

Comment:
Would it be possible to have more than one persons in management-list? And why is the first one set as "LineOrgResponsible"?

Reply:
Yes. This is a short term quick fix, and should work for most departments. However we need to address handling multiple managers, but this must also be supported in the UI. As there is only one property on the api model.